fv3core/pace/fv3core/testing/translate_fvdynamics.py

```diff
@@ -282,6 +282,8 @@ def __init__(
         self._base.out_vars.update(fv_dynamics_vars)
         self._base.out_vars["ps"] = {"kstart": grid.npz - 1, "kend": grid.npz - 1}
         self._base.out_vars["phis"] = {"kstart": grid.npz - 1, "kend": grid.npz - 1}
+        self._do_not_subset_tracers = True
```

Comment:
See comment below about isolating this logic in this class so this private attribute isn't read by other objects.